src/rp2_common/hardware_rtc/include/hardware/rtc.h

```diff
@@ -32,6 +32,10 @@
 extern "C" {
 #endif
 
+#ifndef PARAM_ASSERTIONS_ENABLED_RTC
```

Comment:
Could you add a PICO_CONFIG line as well please? See https://github.com/raspberrypi/pico-sdk/blob/master/src/rp2_common/hardware_adc/include/hardware/adc.h#L47 for an example.
(This allows the value to be set by https://github.com/raspberrypi/pico-project-generator )

Reply:
done